docs/swagger.yaml

```diff
@@ -455,11 +525,11 @@ paths:
       required: true
       type: "string"
       description: "The charge identifier (id)"
-  /api/v1/draftAgreement:
+  /api/v1/agreements/draft:
     post:
       tags:
       - "draft-agreement-controller"
-      summary: "Send a new customer to Vipps in order to accept Agreement"
+      summary: "Create a new Agreement"
```

Comment:
Make sure we tell them that the customer is involved

Reply:
"Create a new Agreement, to be confirmed in Vipps"?